cpp/src/parquet/statistics_test.cc

```diff
@@ -459,6 +459,10 @@ class TestStatistics : public PrimitiveTypedTest<TestType> {
     EXPECT_TRUE(enc_stats->has_max);
     EXPECT_EQ(expected_stats->EncodeMin(), enc_stats->min());
     EXPECT_EQ(expected_stats->EncodeMax(), enc_stats->max());
+    EXPECT_TRUE(enc_stats->is_min_value_exact.has_value());
+    EXPECT_TRUE(enc_stats->is_max_value_exact.has_value());
+    EXPECT_TRUE(enc_stats->is_min_value_exact);
+    EXPECT_TRUE(enc_stats->is_max_value_exact);
```

Comment:
You might be able to condense this to:
```suggestion
    EXPECT_EQ(enc_stats->is_min_value_exact, std::make_optional(true));
    EXPECT_EQ(enc_stats->is_max_value_exact, std::make_optional(true));
```